test2code/src/main/kotlin/com/epam/drill/test2code/Test2Code.kt

```diff
@@ -106,14 +106,21 @@ class Test2Code(
 
     override fun scanClasses(consumer: (Set<EntitySource>) -> Unit) {
         val packagePrefixes = configuration.agentMetadata.packagesPrefixes
-        val additionalPaths = configuration.parameters[ParameterDefinitions.SCAN_CLASS_PATH]
+        val scanClassPaths = configuration.parameters[ParameterDefinitions.SCAN_CLASS_PATH]
+        val isScanClassLoaders = configuration.parameters[ParameterDefinitions.IS_SCAN_CLASS_LOADERS]
```

Comment:
I suggest to rename to `ENABLE_SCAN_CLASS_LOADERS`, otherwise its very counterintuitive